src/main/resources/com/google/api/codegen/php/discovery_fragment.snip

```diff
@@ -31,99 +33,176 @@
     @end
 
     {@body}
-
   @end
 @end
 
 @snippet generateBody(method)
   @let apiName = context.getSimpleApiName, \
-       serviceName = context.getServiceClassName, \
-       params = context.getFlatMethodParams(method)
-    // Create a new client.
+       serviceName = context.getServiceClassName
+    // Authentication is provided by the `gcloud` tool when running locally
+    // and by built-in service accounts when running on GAE, GCE, or GKE.
+    // For more information, see
+    // https://developers.google.com/identity/protocols/application-default-credentials
     $client = new Google_Client();
-    $client->setApplicationName('Client Sample Application');
+    $client->setApplicationName('Google Cloud Platform Sample');
     $client->useApplicationDefaultCredentials();
     $client->addScope('https://www.googleapis.com/auth/cloud-platform');
 
-    // Create a new {@apiName} service.
+    // Construct the {@apiName} service object for interacting with the API.
     $service = new {@serviceName}($client);
-    {@initialization(method, params)}
 
+    {@initialize(method)}
     @if context.isPageStreaming(method)
-      {@iterableCall(method, params)}
+      {@iterableCall(method)}
     @else
-      {@compactCall(method, params)}
+      {@compactCall(method)}
     @end
   @end
 @end
 
-@private initialization(method, params)
+@private initialize(method)
   @let methodName = context.getMethodName(method), \
-       apiaryConfig = context.getApiaryConfig, \
-       signatureType = apiaryConfig.getType(method.getRequestTypeUrl)
-    @join param : params
-      @let paramField = context.getField(signatureType, param), \
-           paramValue = context.typeDefaultValue(signatureType, paramField), \
-           paramDescription = apiaryConfig.getDescription(signatureType.getName, param)
-
-        {@description(paramDescription)}
-        ${@param} = {@paramValue};
+       signatureType = context.getApiaryConfig.getType(method.getRequestTypeUrl), \
+       params = context.getFlatMethodParams(method)
+    @if params
+
+      // {@TODO()} Change placeholders below to desired parameter values for the `{@methodName}` method:
+
+      @join param : params
+        @let paramField = context.getField(signatureType, param), \
+             paramValue = context.typeDefaultValue(signatureType, paramField), \
+             paramDescription = context.getDescription(signatureType.getName, param)
+          {@description(paramDescription)}
+          ${@param} = {@paramValue};
+
+        @end
       @end
-    @end
 
-    @if context.hasRequestField(method)
+    @end
+  @end
+  @if context.hasRequestField(method)
 
-      $postBody = new {@context.getRequestClassName(method)}($client);
+    ${@requestBodyName(method)} = new {@context.getRequestClassName(method)}($client);
```

Comment:
I see this name used in internal client library generation code, but I don't find it in documentation anywhere, and it's a very generic term. I feel the informative variable name is preferable here.